packages/cli/src/commands/dns/index.ts

```diff
@@ -38,6 +38,9 @@ const help = () => {
   )}        Login token
     -S, --scope                    Set a custom scope
     -N, --next                     Show next page of results
+    --limit=${chalk.bold.underline(
+      'VALUE'
+    )}                       Number of results to return per page (default: 20, max: 100)
```

Comment:
Likewise, I think there are 5 too many spaces here between the option and description.